package.json

```diff
@@ -123,6 +123,7 @@
     "balena-semver": "^2.3.0",
     "balena-settings-client": "^4.0.6",
     "date-fns": "^2.29.3",
+    "handlebars": "^4.7.7",
```

Comment:
Do you know how much size this adds to the sdk bundle?

Reply:
looks like it adds 0.2Mb

Reply:
I wonder whether we could rely on `_.template`, but we can change that later if we need to investigate & drop the size increase.

Reply:
Not changing this and can be discussed later.